### DISCUSSION TITLE
Example doesn't seem to work: Module  "vike/types"  has no exported member  OnRenderClientAsync 

### DISCUSSION THREAD

**Discussion by crummy** [vikejs/vike#1199] [Discussion] [answered]
1. Clone vike repo
2. Open examples/react-full-v1
3. npm install
4. Run ` ./node_modules/.bin/tsc --noEmit`

A bunch of errors appear, including: 

```
renderer/+onRenderHtml.tsx:9:15 - error TS2305: Module '"vike/types"' has no exported member 'OnRenderHtmlAsync'.
```

**Comment by brillout** [comment, marked as answer]
It's because the hook types such as `OnRenderHtmlAsync` aren't released yet. We will release them today. In the meantime, use `pnpm` in order to use the local copy of Vike instead of fetching it from npm (let me know if you want more detailed instructions).

**Reply by lourot** [reply]
(released, should work again)
